src/service/data_products/genome_attributes.py

```diff
@@ -451,12 +499,84 @@ async def get_genome_attributes(
     }
 
 
+class Histogram(BaseModel):
+    
+    bins: Annotated[list[float], Field(
```

Comment:
bins should be `list[int]` and values should be `list[float]` right?

Reply:
bins is a list of float. Let's say [2.5, 3.5, 4.5], then we have [2.5, 3.5), [3.5, 4.5), and (4.5, upperbound]
values should be a list of int, indicating how many counts in that particular bin.

Reply:
👍 I see. the `example ` appears conflicting.

Reply:
fixed